lib/measured/conversion.rb

```diff
@@ -3,6 +3,7 @@ class Measured::Conversion
   def initialize(case_sensitive: false)
     @base_unit = nil
     @units = []
+    @unit_name_to_unit = {}
```

Comment:
Whenever a unit is added, we also add a mapping from its name and aliases to this mapping. This allows us to quickly decide if a given name has an associated unit.